components/style/values/computed/mod.rs

```diff
@@ -82,6 +84,8 @@ pub mod flex;
 pub mod font;
 pub mod image;
 #[cfg(feature = "gecko")]
```

Comment:
This doesn't have anything Gecko-specific, right? Let's just build it for Servo too so we have less divergence.

Reply:
Right, I think I thought that might trigger warnings, but quite possibly I thought wrong!